### PR TITLE
Feature member

### PR DESCRIPTION
**Pull request by pacyL2K19**
# Description of  this PR 

This PR contains 
- The data model of :
> Users
> Post
- The corresponding controllers 
- The MVP UIs for registration, authentication, and home page
- The code linters

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  before_action :authenticate_user!, except: %i[index show]
+
+  def index
+    @posts = Post.all.order('created_at DESC')
+  end
+
+  def new
```
Comment (GzimAsani) [inline app/controllers/posts_controller.rb:8]: it should initialize an empty post `@post = Post.new` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_new_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :red_circle: 

### To highlight:
- Neat project
- I like the layout/colors of your page, good job :muscle: 

### To imprve:
- [x] In **Posts/Controller** you need to create a custom before_action method to restrict users to specific actions and that custom method should be declared in ApplicationController  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/before_action_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)). This functionality might have been covered by before_action :authenticate_user! but it is very important that you learn on how to create your own!
- [x] Commit messages should always be written in the present tense and should start with a capital letter ex Add Example Method. You can change the ones written in the past tense using git rebase

Please fix the required changes and ask for another code review whenever you're ready

_**If you have any question related to the required changes feel free comment here, or ping me on slack @GzimAsani**_

HAPPY CODING :computer: :desktop_computer:

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello Felipe Angulo Torres & Pacifique Linjanja** 
Amazing job :tada: :tada: :tada:  you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the develop branch
 
Keep going like this  :clap: :clap:
 
Oscar Andrés Russi Porras

**Comment by pipe2442** [comment]
Thanks!!! @andresporras3423

**Pull request merged**